Cannot import `nitro-protocol`

**Describe the bug :bug:**

Cannot import `nitro-protocol` on a [published canary build](https://github.com/statechannels/statechannels/actions/runs/483828683) based on `master`.

```
@graphprotocol/statechannels:     Cannot find module 'lodash.pick' from '../../node_modules/@statechannels/nitro-protocol/lib/src/index.js'
@graphprotocol/statechannels:     Require stack:
@graphprotocol/statechannels:       /root/project/node_modules/@statechannels/nitro-protocol/lib/src/index.js
@graphprotocol/statechannels:       /root/project/node_modules/@statechannels/wallet-core/lib/src/state-utils.js
@graphprotocol/statechannels:       /root/project/node_modules/@statechannels/wallet-core/lib/src/index.js
@graphprotocol/statechannels:       src/utils/app-data.ts
@graphprotocol/statechannels:       test/attestationApp.test.ts
@graphprotocol/statechannels:       at Resolver.resolveModule (../../node_modules/jest-resolve/build/index.js:306:11)
@graphprotocol/statechannels:       at Object.<anonymous> (../../node_modules/@statechannels/nitro-protocol/src/index.ts:1:1)
@graphprotocol/statechannels:       at Object.<anonymous> (../../node_modules/@statechannels/wallet-core/src/state-utils.ts:1:1)
```


**To Reproduce**
https://app.circleci.com/pipelines/github/statechannels/the-graph/1941/workflows/69833477-8889-41fd-9cc4-843a51884f45/jobs/8272

--- a/packages/nitro-protocol/package.json
+++ b/packages/nitro-protocol/package.json
@@ -8,7 +8,9 @@
   "dependencies": {
     "@openzeppelin/contracts": "3.2.2-solc-0.7",
     "ethers": "5.0.12",
-    "lodash.isequal": "^4.5.0"
+    "lodash.isequal": "^4.5.0",
+    "lodash.pick": "4.4.0",
+    "lodash.shuffle": "^4.2.0"
   },
   "devDependencies": {
     "@nomiclabs/hardhat-etherscan": "^2.1.0",

